src/test/bundler_test.ts

```diff
@@ -177,14 +177,36 @@ suite('Bundler', () => {
   });
 
   test('lazy imports are not moved', async () => {
-    assert.equal(
-        dom5.queryAll(
-                await bundle('test/html/imports/lazy-imports.html'),
-                preds.AND(
-                    preds.parentMatches(preds.hasTagName('dom-module')),
-                    matchers.htmlImport))
-            .length,
-        2);
+    const bundler = new Bundler({
+      analyzer:
+          new Analyzer({urlLoader: new FSUrlLoader('test/html/imports')})
+    });
+    const manifest = await bundler.generateManifest(['lazy-imports.html']);
+    const documents = await bundler.bundle(manifest);
+    const matchLazyImportInsideDomModule = preds.AND(
+        preds.parentMatches(preds.hasTagName('dom-module')),
+        matchers.htmlImport);
```

Comment:
I think *I* was lazy here.  The matcher here isn't even used until after the first query, so this is defined prematurely, and also, the fact that it matches all html imports isn't clear from the name, which really should just be "matchHtmlImportsInDomModule".  I'm getting rid of the variable altogether and being specific in the actual query now.